scripts/hal_alignment/hal_gene_liftover.py

```diff
@@ -58,7 +58,12 @@ def make_src_region_file(regions: Iterable[Union[pybedtools.cbedtools.Interval,
         bed_file: Path of BED file to output.
         flank_length: Length of upstream/downstream flanking regions to request.
 
+    ValueError: If the flank length is negative.
+
     """
+    if flank_length < 0:
+        raise ValueError(f'Flank length must be greater than or equal to 0: {flank_length}')
```

Comment:
[Suggestion] I'm usually prefer writing the variable name, so the user knows directly which one you are talking about:
```suggestion
        raise ValueError(f"'flank_length' must be greater than or equal to 0: {flank_length}")
```